src/main/res/values/dimens.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
 
-    Copyright 2015 Red Hat, Inc. and/or its affiliates
+    Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
```

Comment:
Mistake in the fuss.
command modified it somehow and i didn't noticed.
Added by mistake with rest of commits